src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -401,8 +401,30 @@ private SimulationParameterSwap mapSimulationParameter(SimulationParameterRef si
       private IReadOnlyList<SimulationPlot> retrieveProjectPlots(dynamic reportConfiguration) =>
          GetListFrom<SimulationPlot>(reportConfiguration.Plots?.AllPlots);
 
-      private IReadOnlyList<Input> retrieveInputs(dynamic reportConfiguration) =>
-         GetListFrom<Input>(reportConfiguration.Inputs);
+      private IReadOnlyList<Input> retrieveInputs(dynamic qualificationPlan)
+      {
+         var sections = retrieveSections(qualificationPlan);
+         var inputs = GetListFrom<Input>(qualificationPlan.Inputs);
+         foreach (var input in inputs)
+         {
+            input.SectionLevel = getSectionLevel(sections, input.SectionId);

```

Comment:
I am saving this information in the input. This will allow PKSim to generate the markdown at the expected level